.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: release
+on:
+  push:
+    branches:
+      - main
+      - 'release-*'
```

Comment:
Do we need to run this on pushes to these branches if the check will only happen when a tag is pushed to the repo?

Reply:
I thought I read the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore) to say that if both were defined, both filters would have to be satisfied.  But now I see at the end of the paragraph where they're talking about the various combinations of tags & branches it all of a sudden mentions both branches and *paths* instead of branches and *tags*.  I think that must be a copy/paste error.

Having said that, I have been able to test this by just pushing a tag to my own branch, so it doesn't appear to me that the branches piece is actually doing anything.  I knew that but thought it wouldn't hurt to leave them in.  I'll take out the branches.